lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -264,4 +266,80 @@ describe('ReportUIFeatures', () => {
       assert.ok(container.querySelector('.lh-metrics-toggle__input').checked);
     });
   });
+
+  describe('tools button', () => {
```

Comment:
would also be nice to a get a test for skipping those disabled states since that's a majority of the selection filtering

Reply:
Good point, I will add a unit tests for the filter logic tomorrow. Thanks!

Reply:
Done!